octobot_commons/authentication.py

```diff
@@ -31,6 +32,9 @@ def __init__(self):
         )
         self.initialized_event: asyncio.Event = None
         self.supports: None
+        self.feed_callbacks = {}
+        # also register this instance for the base Authenticator class ion singleton
+        singleton.Singleton._instances[Authenticator] = self
```

Comment:
Why? It should be done using `Authenticator.instance` when calling Authenticator for the first time

Reply:
The reason here is that we don't instanciate an `Authenticator` but a subclass of it however we want to access the subclass instance when calling `Authenticator.instance` since `Authenticator` is abstract.

Reply:
Right, understood 👍